### PR TITLE
Add an example to the top of README

### PR DESCRIPTION
**Pull request by ilyalesik**
@lttb What are you think about add to the top of README some simple example?

### CODE REVIEW

**Comment by lttb** [comment]
this is a great idea, thank you! would you also like to consider something like this?

```jsx
import React from 'react'
import styled from 'reshadow'

export const Button = ({
    color,
    children,
    active,
    ...props,
}) => styled`
    button {
        background-color: white;
        color: ${color};

        &:active, &[aria-pressed] {
            border-color: white;
        }
    }
`(
    <button {...props} aria-pressed={active}>
        {children}
    </button>
)
```

there are some things that I think important to show:
- familiar styled-components interface
- dynamic values
- attributes and state mapping with also aria attributes

**Comment by ilyalesik** [comment]
My idea is to show in each example only one new feature. At first example too. I think at first example it can be "describe whole component elements styles at single tagged template literal" as the main concept. We can add a section "Features" at the next step, and the first feature can be "modifiers". Documentation can move from simple to advanced concepts.

File: README.md [hunk 1]
```diff
@@ -1,12 +1,38 @@
...
+  span {
+    font-size: 14px;
+  }
+`;
+
+export const Button = ({children, onClick}) => {
+  // connect the styles to the markup:
+  return styled(styles)(
+    <button onClick={onClick}>
+      <span>{children}</span>
```
Comment (lttb) [inline README.md]: would not it be a better to use an example like `<content as="span">{children}</content>`?

**Comment by ilyalesik** [comment]
🎉

File: README.md (hunk 1 above)
Comment (ilyalesik) [reply README.md]: Yeah, it's better

**Pull request merged**